packages/fluentui/react-bindings/src/icons/createSvgIcon.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+
+import { StylesContextValue } from '../styles/types';
+import useStyles from '../hooks/useStyles';
+import getUnhandledProps from '../utils/getUnhandledProps';
+import { SvgIconCreateFnParams, SvgIconProps } from './types';
+
+export const SvgIconClassName = 'ui-icon';
+export const SvgIconDisplayName = 'SvgIcon';
+
+export const SvgIconHandledProps: (keyof SvgIconProps)[] = [
+  'alt',
+  'aria-label',
+  'bordered',
+  'className',
+  'circular',
+  'children',
+  'design',
+  'disabled',
+  'outline',
+  'size',
+  'rotate',
+  'styles',
+  'variables',
+  'xSpacing',
+];
+
+const createSvgIcon = <TProps extends SvgIconProps>({
+  svg,
+  displayName,
+  handledProps = SvgIconHandledProps,
```

Comment:
```suggestion
  handledProps = []
```

I suggest this and then:

```
const unhandledProps = getUnhandledProps([ ...SvgIconHandledProps, ...handledProps], props);
```

Because otherwise I will need to merge this in `createSvgIcon` by myself 😭

Reply:
Fixed